addOns/help/src/main/javahelp/contents/credits.html

```diff
@@ -148,6 +148,7 @@ <H2>ZAP Extended Team</H2>
 
 <tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Bunyanuch Saengnet</td></tr>
 <tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Goran Sarenkapa (JordanGS)</td></tr>
+<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Pranav Saxena (<a href="https://github.com/pranavsaxena17">@pranavsaxena17</a>)</td></tr>
```

Comment:
Should be after the next line?

Reply:
Yup I think you're right. I saw the "Si" and made a poor assumption.